students/charlie_england/session04/dict_lab.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+#Dictionaries 1:
+first_dict = {"name":"Chris","city":"Seattle","cake":"Chocolate"}
+print(first_dict)
+del first_dict["cake"]
+print(first_dict)
+first_dict.update({"fruit":"Mango"})
+print(first_dict.keys())
+print(first_dict.values())
+if "cake" not in first_dict.keys():
```

Comment:
When searching keys in a dict, you don't need to specify the dict.keys()
ex: if "cake" not in first_dict 
This will return the same logic